azure-pipelines.yml

```diff
@@ -820,6 +820,8 @@ jobs:
   - bash: |
       set -ev
       . env/bin/activate
+
+      pip install --force -q git+https://github.com/Azure/azure-cli-dev-tools.git@master
```

Comment:
This is the different between this PR and https://github.com/Azure/azure-cli/pull/14402/